index.html

```diff
@@ -2,9 +2,29 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Hello World</title>
+    <title>Code Coop</title>
+    <link rel="stylesheet" href="assets/css/index.css">
   </head>
   <body>
-      <h1>Hello World!</h1>
+    <header>
+        <h1>Code Coop</h1>
+    </header>
+      <nav>
+        <a href="about.html">About</a>
+        <a href="contact.html">Contact Us</a>
```

Comment:
Add a title attribute https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#Adding_supporting_information_with_the_title_attribute

Reply:
This is a link to a file that doesn't exist yet (i.e. about.html). Would be good to add it now, even though it doesn't actually have content.